src/type/CustomField.ts

```diff
@@ -72,7 +72,33 @@ export const toJson = (
       })
       // Collapse all the JSON object entries into a single object.
       .reduce(
-        (acc, cur) => Object.assign(acc, cur),
```

Comment:
This change makes it so that if the provided custom fields contain the same key multiple times, values are accumulated in an array rather than only overidden to only consider the latest one.